expo/tests/gbe/test_landing_page.py

```diff
@@ -1,97 +1,215 @@
+                                                 submitted=True,
+                                                 accepted=3)
+        self.current_class.title = "Current Class"
+        self.current_class.conference = self.current_conf
+        self.current_class.save()
+        self.previous_class = ClassFactory.create(teacher=self.performer,
+                                                  submitted=True,
+                                                  accepted=3)
+        self.previous_class.conference = self.previous_conf
+        self.previous_class.title = 'Previous Class'
+        self.previous_class.save()
+        self.current_vendor = VendorFactory.create(profile=self.profile,
+                                                   submitted=True)
+        self.current_vendor.conference = self.current_conf
+        self.current_vendor.title = "Current Vendor"
+        self.current_vendor.save()
+        self.previous_vendor = VendorFactory.create(profile=self.profile,
+                                                    submitted=True)
+        self.previous_vendor.conference = self.previous_conf
+        self.previous_vendor.title = 'Previous Vendor'
+        self.previous_vendor.save()
+        self.current_costume = CostumeFactory.create(profile=self.profile,
+                                                     submitted=True)
+        self.current_costume.conference = self.current_conf
+        self.current_costume.title = "Current Costume"
+        self.current_costume.save()
+        self.previous_costume = CostumeFactory.create(profile=self.profile,
+                                                      submitted=True)
+        self.previous_costume.conference = self.previous_conf
+        self.previous_costume.title = 'Previous Costume'
+        self.previous_costume.save()
+        self.current_volunteer = VolunteerFactory.create(profile=self.profile,
+                                                         submitted=True)
+        self.current_volunteer.conference = self.current_conf
+        self.current_volunteer.save()
+        self.previous_volunteer = VolunteerFactory.create(profile=self.profile,
+                                                          submitted=True)
+        self.previous_volunteer.conference = self.previous_conf
+        self.previous_volunteer.save()
+        
+        # Event assignments, previous and current
+        current_opportunity = GenericEventFactory.create(
+            conference=self.current_conf,
+            title="Current Volunteering",
+            type='Volunteer')
+        current_opportunity.save()
+        previous_opportunity = GenericEventFactory.create(
+            title="Previous Volunteering",
+            conference=self.previous_conf)
+        previous_opportunity.save()
+
+        self.current_sched = Event(
+            eventitem=current_opportunity,
+            starttime=datetime(2016, 2, 5, 12, 30, 0, 0, tzinfo=pytz.utc),
```

Comment:
Below, you do this without naming the argument. I like that better. Can you change this one to match?